keps/prod-readiness/4427.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4427
+alpha:
+  approver: "@thockin"
```

Comment:
sorry, I am not PRR approver - @jpbetz @johnbelamaric @wojtek-t or @deads2k  - can one of you serve?  This one should be easy.

Reply:
Yes, go ahead and put my GitHub handle here and I'll review.